### DISCUSSION TITLE
Save and load while saving and reloading all attributtes

### DISCUSSION THREAD

**Discussion by roycw** [geoman-io/leaflet-geoman#1500] [Questions & Help]
HI Guys, been using this lib for a while and its amazing.

I am sure someone already did this before but i want to save and relaod my drawing but also relaod all the attributes. for example if i saved a textMArker it should show upas a text marker and not a plaing marker. im sure there is a oiler plate somewheer instyead of itertainig through all layers and have if.. lses everywhere... any help or pointers will be appreciated.
Thanks!

**Comment by Falke-Design** [comment]
Hi @roycw,

you can look into #1182 but we have no boilerplate for that.

**Comment by roycw** [comment]
Thanks a bunch!
